### PR TITLE
Don't add floppy for arm/aarch

### PR DESCRIPTION
**Pull request by cwh42**
Fixing https://progress.opensuse.org/issues/25622

### CODE REVIEW

File: backend/qemu.pm [hunk 1]
```diff
@@ -563,7 +563,9 @@ sub start_qemu {
     {
         push(@params, @vgaoptions);
 
-        gen_params @params, "global", "isa-fdc.driveA=" unless $vars->{QEMU_NO_FDC_SET};
+        unless ($vars->{ARCH} eq 'aarch64' || $vars->{ARCH} eq 'arm' || $vars->{QEMU_NO_FDC_SET}) {
```
Comment (SergioAtSUSE) [inline backend/qemu.pm]: Why `$vars->{ARCH} eq 'arm'`?
I think we don't use this name.

Comment (okurz) [inline backend/qemu.pm]: Please add a comment describing why the parameter does not apply for aarch64 (or arm)

Comment (cwh42) [reply backend/qemu.pm]: Used that to be consistent compared to #L433

Comment (SergioAtSUSE) [reply backend/qemu.pm]: You are right.

This is os-autotinst. We don't use this name in os-autoinst-distri*.

**Comment by cwh42** [comment]
Added an explaining comment to the change.

**Comment by okurz** [comment]
Awesome, LGTM

**Pull request merged**